src/pages/search/advanced_search/OrganizationHierarchyFilter.tsx

```diff
@@ -52,13 +52,19 @@ export const OrganizationHierarchyFilter = ({ organization, open, onClose }: Org
   const closeDialog = () => {
     onClose();
     setSearchId('');
-    setSelectedId(unitFromParams);
```

Comment:
Hvorfor trekke denne ut?

Reply:
Den der resetter valgt verdi til å bli det samme som er i URLen, og det ønsker vi ikke å gjøre når brukeren velger en ny enhet.
Det fører til en bug i dev nå, om man velger en enhet, og så åpner dialogen på nytt. Da ville denne snutten resatt valgt enhet ved en feil

Reply:
forskjellen nå er altså at denne er manuelt lagt til der den skal, mens den er utelatt for submit (linje 125)

Reply:
Ah, jeg må ha oversett linje 125 totalt 🙈 Lette etter en plass hvor det ikke skulle brukes... Da er den god! 👍

Reply:
Ser ut som jeg kun så på den første committen av en eller annen grunn...